site/src/pages/release-notes/1.1.0.mdx

```diff
@@ -0,0 +1,86 @@
+---
+date: 2020-09-18
+---
+
+## 🌟 New features
+
+- You can now inject your own <type://MeterIdPrefixFunction> when using Armeria Spring boot autoconfigure. #3034
+  ```java
+  @Bean
+  public MeterIdPrefixFunction myMeterIdPrefixFunction() {
+      return MeterIdPrefixFunction.of("custom.armeria.server");
+  }
+  ```
+- You can now use a string to specify the `Logger` for <type://LoggingService> and <type://LoggingClient>. #3035
+  ```java
+  ServerBuilder sb = ...
+  sb.decorator(LoggingService.builder()
+                             .logger("com.foo")
+                             .build());
+  ```
+
+## 📈 Improvements
+
+- Span name is the service name if you use the default `HttpRequestParser` and `HttpResponseParser` with
+  `BraveClient` and `BraveService`. #3023
+- You can now easily find out whether SPI service interfaces are applied from the logs. #3039
+- You can now get the log of the exception which is raised while making request and response logs. #3043
+- The stack-trace of <type://HttpStatusException> and <type://HttpResponseException> are not logged in
```

Comment:
- stack-trace -> stack trace
- not logged anymore by 
- Explain why we don't do that anymore. Depending on the explanation, could it be a bug fix?